_data/simple-icons.json

```diff
@@ -1450,6 +1450,11 @@
             "source": "https://www.aquasec.com/brand",
             "guidelines": "https://www.aquasec.com/brand"
         },
+        {
+            "title": "Arabica Coffee",
+            "hex": "231815",
+            "source": "https://upload.wikimedia.org/wikipedia/commons/b/b7/%25_Arabica_percent_logo.svg"
```

Comment:
If you're using Wikipedia as a source, you must link to the Wikimedia Commons version of the image. If that does not exist, we cannot use that image.